framework/filters/PageCache.php

```diff
@@ -123,6 +124,13 @@ class PageCache extends ActionFilter
      * @since 2.0.4
      */
     public $cacheHeaders = true;
+    /**
+     * @var array a list of placeholders for embedding dynamic contents. This property
+     * is used internally to implement the content caching feature. Do not modify it.
+     * @internal
+     * @since 2.0.11
+     */
```

Comment:
this naming is a bit confusing as it look like the placeholders are dynamic. I'd prefer something like "dynamicContentPlaceholders"